hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/ClientNearCacheExecutor.java

```diff
@@ -24,14 +24,14 @@
 
 public final class ClientNearCacheExecutor implements NearCacheExecutor {
```

Comment:
Imho this class is just noise. Define a single ScheduledExecutor interface with scheduleWithRepetition and offer it on a 'util' level. Then we don't need to make all these wrapper classes.